### ISSUE TITLE
Environment variables are not honored

### ISSUE THREAD

**Issue by lwbt** [flathub/org.jellyfin.JellyfinServer#87]
I wanted to run the Flatpak version of Jellyfin server in headless mode and found that environment variables are not honored. To run in headless mode one is supposed to set `$JELLYFIN_SERVICE_OPT` to `"--service"`.

- https://github.com/jellyfin/jellyfin/blob/3e9bead6972a52db0a82f517590fed7fe0db3a4f/debian/conf/jellyfin#L28C23-L28C34
- https://github.com/jellyfin/jellyfin/blob/3e9bead6972a52db0a82f517590fed7fe0db3a4f/debian/jellyfin.service#L11C31-L11C140

This is unexpected for users who want to migrate from DEB/RPM to Flatpak. I want to suggest calling the binary with the same parameters as it is done in the upstream repository for Debian and Fedora. And also to not try to start a browser.

My suggestion would be to make the following changes:

```bash
--- org.jellyfin.JellyfinServer.sh	2023-08-12 23:57:11.115058971 +0200
+++ org.jellyfin.JellyfinServer_.sh	2023-08-13 00:12:29.853602245 +0200
@@ -3,9 +3,11 @@
 PORT=8096
 
 if ! curl http://localhost:$PORT > /dev/null 2>&1; then
-  (sleep 10 && xdg-open http://localhost:$PORT) &
+  if [ -z "$JELLYFIN_SERVICE_OPT" ]; then
+    (sleep 10 && xdg-open http://localhost:$PORT) &
+  fi
   mkdir -p ~/.var/app/org.jellyfin.JellyfinServer/media
-  jellyfin
+  jellyfin $JELLYFIN_WEB_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT $JELLYFIN_ADDITIONAL_OPTS
 else
   xdg-open http://localhost:$PORT
 fi
```

I wanted to suggest using `! "${JELLYFIN_SERVICE_OPT}" =~ "--service"` until I realized that the shebang is `#!/bin/sh`.

It should be possible for users to adapt the existing systemd service units with minor changes and have the flatpaked server run without manual intervention. When I tried, systemd said it received exit code 137.

**Comment by istori1** [comment]
Please submit a pull request with the changes. Thank you.

**Comment by lwbt** [comment]
This will be implemented with the improved launcher of the v10.10 releases.

https://github.com/flathub/org.jellyfin.JellyfinServer/pull/432/files#diff-446f173b83737aa24ae61df0be9c80e3fd8c1dbcafd64b2faa0e1fa767c80923
